cookiecutter-xblock/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -59,8 +59,14 @@ dev.run: dev.clean dev.build ## Clean, build and run test image
 
 extract_translations: symlink_translations ## extract strings to be translated, outputting .po files
 	cd $(PACKAGE_NAME) && i18n_tool extract
```

Comment:
@shadinaif It seems that I've been bringing this up every time I see `i18n_tool` is used. Can we _not_ use it by default and stick with the default `django-admin makemessages`?

Otherwise, this workflow should be bundled within the `i18n_tool` itself so we don't have to copy ~12 lines of code all over the XBlocks.

Reply:
Thank you @OmarIthawi . I've created a PR for the second option https://github.com/openedx/i18n-tools/pull/134 . I'll ping you there when it's ready